core/simpleproducer.go

```diff
@@ -123,6 +125,19 @@ func (prod *SimpleProducer) Configure(conf PluginConfigReader) error {
 	dropStreamID := StreamRegistry.GetStreamID(conf.GetString("DropToStream", DroppedStream))
 	prod.dropStream = StreamRegistry.GetRouterOrFallback(dropStreamID)
 
+	// Simple health check for the plugin state
+	//   Path: "/<plugin_class>/<plugin_id>/SimpleProducer/pluginstate"
```

Comment:
As pluginID has to be unique over all plugin types we don't really need the plugin_class.
So it should be /<plugin_id>.
Simple producer may include a basic health check like this, but I would prefer function chaining, i.e. creating a list of functions to call as a health check and combining their results (TBD)